source/libs/icons.tsx

```diff
@@ -52,6 +52,9 @@ export const gear = (): JSX.Element => <svg aria-hidden="true" className="octico
 
 export const foldDown = (): JSX.Element => <svg aria-hidden="true" className="octicon octicon-fold-down" width="14" height="16" ><path fill-rule="evenodd" d="M4 11l3 3 3-3H8V5H6v6H4zm-4 0c0 .55.45 1 1 1h2.5l-1-1h-1l2-2H5V8H3.5l-2-2H5V5H1c-.55 0-1 .45-1 1l2.5 2.5L0 11zm10.5-2H9V8h1.5l2-2H9V5h4c.55 0 1 .45 1 1l-2.5 2.5L14 11c0 .55-.45 1-1 1h-2.5l1-1h1l-2-2z"/></svg>;
 
+// The viewBox is actually needed here
```

Comment:
@bfred-it Why are we not using `viewBox` on all of the icon? It let's use resize any icon while preserving aspect ratio.

Reply:
Because a year ago we realized it wasn’t needed on any but a few, so we dropped the “cruft”